src/dislocker.rb

```diff
@@ -11,10 +11,12 @@ class Dislocker < Formula
   version "0.4.0"
 
   depends_on "polarssl"
-  depends_on "osxfuse"
+  depends_on :osxfuse
 
   def install
+    system "mkdir -p #{prefix}/{bin,lib,share/man/man1}"
     system "make -C src"
     system "make -C src install DESTDIR=#{prefix}/"
+    system "cp #{prefix}/lib/libdislocker.0.dylib #{prefix}/bin/"
```

Comment:
aaron-mbp:bin aaron$ ./dislocker
dyld: Library not loaded: libdislocker.0.dylib
  Referenced from: /usr/local/Cellar/dislocker/0.4.0/bin/./dislocker
  Reason: image not found

Copying the file from lib (it did exist there) to bin sorted the above error.

I couldn't bother with figuring out why files in lib weren't being found or why so many symlinks were made within the lib directory itself.. Perhaps it was erroneously making symlinks which were meant to link the file to bin or some folder dislocker checks for includes. Why else would it symlink the file to another file in the same lib folder.. make install output below:

aaron-mbp:src aaron$ make install
install -pm755  dislocker-fuse  dislocker-file  dislocker-metadata  dislocker-bek /usr/local/Cellar/dislocker/0.4.0/bin/
ln -s dislocker-fuse /usr/local/Cellar/dislocker/0.4.0/bin/dislocker
install -pm755 libdislocker.0.dylib /usr/local/Cellar/dislocker/0.4.0/lib/
ln -s /usr/local/Cellar/dislocker/0.4.0/lib/libdislocker.0.dylib /usr/local/Cellar/dislocker/0.4.0/lib/libdislocker.dylib.0
ln -s /usr/local/Cellar/dislocker/0.4.0/lib/libdislocker.dylib.0 /usr/local/Cellar/dislocker/0.4.0/lib/libdislocker.dylib
cp ../man/dislocker_osx_man ../man/dislocker.1
gzip ../man/dislocker.1
../man/dislocker.1.gz already exists -- do you wish to overwrite (y or n)? y
mv ../man/dislocker.1.gz /usr/local/Cellar/dislocker/0.4.0/share/man/man1/
===========================================================
'dislocker' installed into /usr/local/Cellar/dislocker/0.4.0/bin/
See 'man 1 dislocker' for details on how to use it